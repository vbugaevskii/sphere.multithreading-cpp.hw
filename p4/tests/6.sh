echo trap 'echo sigint ; exit' INT ; kill -INT $PPID ; sleep 0.1 ; echo still alive | bash 
echo done
