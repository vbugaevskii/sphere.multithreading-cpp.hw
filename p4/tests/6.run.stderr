Process 13981 exited: 0
Process 13982 exited: 0
Process 13984 exited: 0
